### PR TITLE
fix readme error

### PR DESCRIPTION
**Pull request by zhichengliu12581**
fix error dependency and missed link

### CODE REVIEW

**Review by alzimmermsft — APPROVED** [review]

**Comment by chenrujun** [comment]
Hi, @lzc-1997-abel , please resolve the conflicts.

**Comment by lzc-1997-abel** [comment]
@chenrujun resolved

File: sdk/appconfiguration/spring-cloud-starter-azure-appconfiguration-config/README.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ There are two libraries that can be used spring-cloud-azure-appconfiguration-con
 
 or
 
-[//]: # ({x-version-update-start;com.microsoft.azure:spring-cloud-azure-appconfiguration-config;current})
+[//]: # ({x-version-update-start;com.microsoft.azure:spring-cloud-azure-appconfiguration-config-config-web;current})
```
Comment (chenrujun) [inline sdk/appconfiguration/spring-cloud-starter-azure-appconfiguration-config/README.md:26]: Duplicated `-config`, please delete one.

File: sdk/appconfiguration/spring-cloud-starter-azure-appconfiguration-config/README.md [hunk 2]
```diff
@@ -23,11 +23,11 @@ There are two libraries that can be used spring-cloud-azure-appconfiguration-con
...
 ```xml
 <dependency>
     <groupId>com.microsoft.azure</groupId>
-    <artifactId>spring-cloud-azure-appconfiguration-config</artifactId>
+    <artifactId>spring-cloud-azure-appconfiguration-config-config-web</artifactId>
```
Comment (chenrujun) [inline sdk/appconfiguration/spring-cloud-starter-azure-appconfiguration-config/README.md:30]: Same here.

**Review by chenrujun — CHANGES_REQUESTED** [review]

**Review by chenrujun — APPROVED** [review]
LGTM.

**Pull request merged**